### PR TITLE
Updated styling for Import Modal

### PR DESCRIPTION
**Pull request by arashveysi3**
fixed a styling problem that keeps happening when you click on 'settings/import from file' in light mode. stylings changed to drop blur instead of white and black with an opacity of %10

**What does this PR do?**
The import screen dialog background is now blurry

**Related Issue(s)**
Issue #383 

**Screenshots or GIFs (if applicable)**
Light Mode:

image

Dark Mode:

Screenshot (6)

### CODE REVIEW

**Comment by arashveysi3** [comment]
It seems I don’t have authorization for deployment checks. please let me know if there’s anything else I should address.

**Comment by bryanlundberg** [comment]
Hi, @arashveysi3 thanks for your interest, today I was going to refactor that modal with a custom dialog from shadcn, that's why I didn't describe it too much 🫡. Because that one continue having the styles from old UI, I just forgot migrate that component.

Anyway, about your implementation, I prefer a dark background instead of a blur. Require a little bit of borders. Also, there is a wrong synchronization showing the blur effect and the modal. Like a delay. Feels strange as user experience.

I'm open to let you refactor that component using shadcn/dialog. ¿Do you want to continue?😅

**Comment by arashveysi3** [comment]
Hey! 
Thanks for reviewing my PR. 
And YES. I am open to any contributions that might help. 
Count on me 🙂

**Comment by bryanlundberg** [comment]
> Hey @bryanlundberg  
> Thanks for reviewing my PR. 
> And YES. I am open to any contributions that might help. 
> Count on me 🙂 

Great

**Comment by arashveysi3** [comment]
> > Hey @bryanlundberg
> > Thanks for reviewing my PR.
> > And YES. I am open to any contributions that might help.
> > Count on me 🙂
> 
> Great

refactored to Shadcn/ui. please take a look.
hope it helps...

File: src/components/menu-settings/ImportModal.tsx [hunk 1]
```diff
@@ -20,105 +28,106 @@ export default function ImportModal() {
...
-                />
-                <Image
-                  src={"/timer-logos/cstimer.jpg"}
-                  alt="cstimer logo"
-                  width={64}
-                  height={64}
-                  className="rounded-2xl"
-                  draggable={false}
-                />
+          <Dialog open={importModalOpen} onOpenChange={setImportModalOpen}>
```
Comment (bryanlundberg) [inline src/components/menu-settings/ImportModal.tsx:34]: `{importModalOpen && (` become redundant with `<Dialog open={importModalOpen}`

File: src/components/menu-settings/ImportModal.tsx [hunk 2]
```diff
@@ -20,105 +28,106 @@ export default function ImportModal() {
   const t = useTranslations("Index.backup-modal");
 
   useClickOutside(componentRef, () => setImportModalOpen(false));
-
   return (
     <>
-      <AnimatePresence>
         {importModalOpen && (
```
Comment (bryanlundberg) [inline src/components/menu-settings/ImportModal.tsx:33]: Makes the fade-out animation from shadcn not be displayed as component disappear from DOM.

File: src/components/menu-settings/ImportModal.tsx [hunk 3]
```diff
@@ -20,105 +28,106 @@ export default function ImportModal() {
   const t = useTranslations("Index.backup-modal");
 
   useClickOutside(componentRef, () => setImportModalOpen(false));
```
Comment (bryanlundberg) [inline src/components/menu-settings/ImportModal.tsx:30]: This can be removed. Not used anymore.

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
Hey @arashveysi3 it's a good improvement, but there are some minor changes before merge. Just to help keep the general components flow


1. the trigger of the modals should be placed in the spot of the button.

That means I need you to change the 

components/menu-settings/DataImportExport.tsx
```
<Dialog>
  <DialogTrigger>Open</DialogTrigger>
  <DialogContentMadeByYou />
</Dialog>
```

2. Your custom dialog for import
```
  <DialogContent>
    <DialogHeader>
      <DialogTitle></DialogTitle>
      <DialogDescription></DialogDescription>
    ....
    </DialogHeader>
  </DialogContent>
```

3. Clean the global state of the previous modal "importModalOpen: boolean" and its function to change. Are not necessary anymore.

store/settingsModalStore.ts

4. Delete the usage of actual `<ImportModal />` app/settings/options/page.tsx line: 129`

I think that's it 👌

File: src/components/menu-settings/ImportModal.tsx (hunk 3 above)
Comment (arashveysi3) [reply src/components/menu-settings/ImportModal.tsx]: Hey! just committed the new improvements. please take a look and let me know if further improvements needed..

**Pull request merged**